opennms-base-assembly/src/main/filtered/etc/datacollection/opennms.appliance.xml

```diff
@@ -0,0 +1,26 @@
+<datacollection-group xmlns="http://xmlns.opennms.org/xsd/config/datacollection" name="OpenNMS Appliances">
+   <systemDef name="OpenNMS Appliances (common)">
+      <sysoidMask>.1.3.6.1.4.1.5813.42.5.</sysoidMask>
+      <collect>
+         <includeGroup>mib2-host-resources-system</includeGroup>
+         <includeGroup>mib2-host-resources-memory</includeGroup>
+         <includeGroup>mib2-host-resources-storage</includeGroup>
+         <includeGroup>mib2-X-interfaces-pkts</includeGroup>
+         <includeGroup>net-snmp-disk</includeGroup>
+         <includeGroup>net-snmp-disk-more</includeGroup>
+         <includeGroup>net-snmp-disk-highlow</includeGroup>
+         <includeGroup>ucd-loadavg</includeGroup>
+         <includeGroup>ucd-memory</includeGroup>
+         <includeGroup>ucd-sysstat</includeGroup>
+         <includeGroup>ucd-sysstat-raw</includeGroup>
+         <includeGroup>ucd-sysstat-raw-more</includeGroup>
+         <includeGroup>ucd-sysstat-more</includeGroup>
+         <includeGroup>ucd-diskio</includeGroup>
+         <includeGroup>ucd-diskio-64bit</includeGroup>
+         <includeGroup>ucd-diskio-load</includeGroup>
+         <includeGroup>lmsensors-temp</includeGroup>
+         <includeGroup>lmsensors-fan</includeGroup>
+         <includeGroup>lmsensors-volt</includeGroup>
```

Comment:
I think the lmsensors MIB tree, might not be supported in the virtual appliance. I think on the Mini or 1U it might work, have not personally tested it.

Reply:
Good point. Even if the objects are present, their values would be meaningless in the case of the virtual appliance. I'll split the systemDef and include lmSensors only for Mini and 1U.